Editor/src/Windows/FileSystemUI.cpp

```diff
@@ -668,27 +680,33 @@ namespace Nuake
                         {
                             for (Ref<Directory>& d : m_CurrentDirectory->Directories)
                             {
-                                if (i + 1 % amount != 0)
-                                    ImGui::TableNextColumn();
-                                else
-                                    ImGui::TableNextRow();
-
-                                DrawDirectory(d, i);
-                                i++;
+                                if(String::Sanitize(d->name).find(String::Sanitize(m_searchKeyWord)) != std::string::npos)
```

Comment:
Not sure if sanitize is necessary? since the filename is already valid for sure

Reply:
ill try to explain it with an example:
lets say i have a folder or file named ‘Test_Folder’ and i type in the search bar ‘ test Folder ’, both will be transformed and compared on the same basis which is ‘testfolder’

Reply:
Awesome